### PR TITLE
Add Changelog for 2.4.0

### PR DESCRIPTION
**Pull request by mpacer**
This updates the changelog for 2.4.0, preparing us for a release later today.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,26 @@
...
+* M Pacer
+* Peter Volpe
+
+The full list of changes they made can be seen [on GitHub](https://github.com/nteract/bookstore/issues?q=milestone%3A2.4.0)
+
+### Significant changes
+
+#### Cloning
+
+As of 2.4.0 cloning from a local or network attached file system is now possible, but disabled by default.
```
Comment (willingc) [inline CHANGELOG.md]: Nice. I like "local or network attached file system".

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,26 @@
...
+
+To enable this fs cloning, set `BookstoreSettings.fs_cloning_basedir` to the root directory from which you want people to be able to clone.
```
Comment (willingc) [inline CHANGELOG.md]:
```suggestion
To enable this filesystem (`fs`) cloning, set `BookstoreSettings.fs_cloning_basedir` to the root directory from which you want people to be able to clone.
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,6 +1,26 @@
...
+
+For more information about the motivation and design of this endpoint, please see [this issue](https://github.com/nteract/bookstore/issues/154).
```
Comment (willingc) [inline CHANGELOG.md]:
```suggestion
Adding fs cloning offers users more flexibility to clone notebooks from attached filesystems, like EFS. For more information about the motivation and design of this endpoint, please see [this issue](https://github.com/nteract/bookstore/issues/154). 
```

File: CHANGELOG.md (hunk 1 above)
Comment (mpacer) [reply CHANGELOG.md]: Thanks!

**Pull request merged**